go/enclave/enclave.go

```diff
@@ -762,9 +781,12 @@ func (e *enclaveImpl) GetTransactionReceipt(encryptedParams common.EncryptedPara
 	}
 	txHash := gethcommon.HexToHash(txHashStr)
 
+	// todo - optimise these calls. This can be done with a single sql
+	e.logger.Trace("Get receipt for ", "txHash", txHash)
 	// We retrieve the transaction.
 	tx, txBatchHash, txBatchHeight, _, err := e.storage.GetTransaction(txHash)
 	if err != nil {
+		e.logger.Trace("error getting tx ", "txHash", txHash, log.ErrKey, err)
```

Comment:
It would be good to have those trace logs with more descriptive messages as our trace logging is very low on context.